Tests/WorkspaceTests/InitTests.swift

```diff
@@ -90,7 +90,7 @@ class InitTests: XCTestCase {
             let triple = Resources.default.toolchain.triple
             let binPath = path.appending(components: ".build", triple.tripleString, "debug")
             XCTAssertFileExists(binPath.appending(component: "Foo"))
-            XCTAssertFileExists(binPath.appending(components: "Foo.build", "Foo.swiftmodule"))
+            XCTAssertFileExists(binPath.appending(components: "Foo.swiftmodule"))
```

Comment:
The feature to allowing linking against executable modules changes the output path based on the tools version, and new packages now get 5.5 by default.  So the unit test changes.